### DISCUSSION TITLE
I have trouble with menu dropdown

### DISCUSSION THREAD

**Discussion by Lilvictoryhh** [kondasoft/ks-bootshop#100] [General]
Hello when i add it a drop down button in the menu, it doesnt work, can u guys help me, its 5.0.0 version just happen in desktop

**Comment by alvinkonda** [comment]
Hello, 

Let me know some more details about this issue. Some screenshots at least if you are not willing to share your store

**Comment by Lilvictoryhh** [comment]
![image](https://github.com/user-attachments/assets/64393a02-a4df-463e-b3dd-f4129598165c)

This is my menu but the 3rd level do not show in the menu 

and this is ther store:https://stylluscuu.myshopify.com/ i hope u guys can help me! thank u

**Comment by alvinkonda** [comment]
Hi,

Our free theme doesn't support a megamenu, unfortunately, but only a dropdown menu, which of course works with 2 levels., so I recommend that you structure your menus with 2 levels only.
